src/api/app/assets/stylesheets/webui2/webui2.css.scss

```diff
@@ -38,6 +38,9 @@
 @import 'live_build_log';
 @import 'borders';
 @import 'diff';
+@import 'staging-workflow';
+@import 'repositories';
```

Comment:
where is 'repositories' coming from?

Reply:
damn. this is another mismerge of bce63b1ddb9651c7c53390be621cbe0d58e37881 - we have repositories twice now